src/graphql/search/filters/utils.ts

```diff
@@ -0,0 +1,28 @@
+import { Filter, FilterOptions } from '../../../generated/graphql'
+
+// Combine both the filters and return the filters
+export function combineFilters(filters: Array<Filter>): Array<Filter> {
```

Comment:
This function is a bit cryptic for me, perhaps it could be split into serpate descriptive methods?